src/main/java/uk/gov/hmcts/darts/datamanagement/dao/impl/DataManagementDaoImpl.java

```diff
@@ -0,0 +1,41 @@
+package uk.gov.hmcts.darts.datamanagement.dao.impl;
+
+import com.azure.storage.blob.BlobClient;
+import com.azure.storage.blob.BlobContainerClient;
+import com.azure.storage.blob.BlobServiceClient;
+import com.azure.storage.blob.BlobServiceClientBuilder;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.darts.datamanagement.config.DataManagementConfiguration;
+import uk.gov.hmcts.darts.datamanagement.dao.DataManagementDao;
+
+import java.util.UUID;
+
+@Component
+@Slf4j
+@RequiredArgsConstructor
+public class DataManagementDaoImpl implements DataManagementDao {
+
+    private final DataManagementConfiguration dataManagementConfiguration;
+    private BlobServiceClient blobServiceClient;
+
+    @Override
+    public BlobContainerClient getBlobContainerClient(String containerName) {
```

Comment:
Is there benefit to exposing `getBlobContainerClient()` as public? It looks like all usages are only calling this to next call `getBlobClient()`.

Wondering if it would make usage easier/cleaner if we simply had `getBlobClient()` as public.

(If motivation is testability, consider making `getBlobContainerClient()` package private instead).